transports/quic/src/transport.rs

```diff
@@ -747,7 +740,7 @@ fn socketaddr_to_multiaddr(socket_addr: &SocketAddr, version: ProtocolVersion) -
 }
 
 #[cfg(test)]
-#[cfg(any(feature = "async-std", feature = "tokio"))]
+#[cfg(any(feature = "tokio"))]
```

Comment:
```suggestion
#[cfg(feature = "tokio")]
```

Here and also in your other PRs please.